src/main/java/org/jabref/gui/preferences/importexport/ImportExportTab.java

```diff
@@ -45,5 +48,9 @@ public void initialize() {
         exportOrderPanel.sortableFieldsProperty().bind(viewModel.sortableFieldsProperty());
         exportOrderPanel.sortCriteriaProperty().bindBidirectional(viewModel.sortCriteriaProperty());
         exportOrderPanel.setCriteriaLimit(3);
+
+        grobidEnabled.selectedProperty().bindBidirectional(viewModel.grobidEnabledProperty());
+        grobidURL.textProperty().bindBidirectional(viewModel.grobidURLProperty());
+        grobidURL.disableProperty().bind(grobidEnabled.selectedProperty().not());
```

Comment:
Could be done in directly in FXML. (`disabled=${grobidEnabled.selected}`) or similar. Lot's of examples elsewhere...

Reply:
I don't think that's possible here because you can't do the negation in fxml. It needs to be disabled if grobidEnabled is NOT selected.